src/components/Login/LoginOTPCard.tsx

```diff
@@ -95,6 +107,22 @@ export const LoginOTPCard: FunctionComponent<LoginOTPCard> = ({
         alert(
           "Encountered an issue obtaining environment information. We've noted this down and are looking into it!"
         );
+      } else if (e instanceof LoginError) {
+        Alert.alert(
```

Comment:
Should Sentry capture the exception here too?

Reply:
hmm that would mean it captures an exception every time someone keys in the wrong otp. which may be very often  haha. it didnt capture previously

Reply:
I see I see, okok, let's leave it then haha